### ISSUE TITLE
storybook/html is affection storybook/react

### ISSUE THREAD

**Issue by neginbasiri** [storybooks/storybook#3980]
If you are reporting a bug or requesting support, start here:
### storybook-html is affecting @storybook/react

I would like to have 2 script files one to run stories for React component and the other to run stories for plain html and scripts. My React storybook works as charm, as soon as I installed storybook-html from npm I get error when trying to run storybook-react task.  

### Steps to reproduce

`
       
        "storybook-react": "start-storybook -p 9001 -c .storybook-react",

        "storybook-html": "start-storybook -p 8001 -c .storybook-html"
`

when I run storybook-react I get bellow 

`
    > ui@1.0.0 storybook-react C:\Projects\LCA\UI
    > start-storybook -p 9001 -c .storybook-react

    info @storybook/html v4.0.0-alpha.11
    info
    Terminate batch job (Y/N)? y
    PS C:\Projects\LCA\UI> info => Loading custom addons config.
    WARN => Custom configuration file C:\Projects\LCA\UI\.storybook-react\webpack.config.js is detected
    WARN    but impossible to import loader for .js
    info => Loading custom webpack config (extending mode).
    10% building modules 2/2 modules 0 activei ｢wdm｣: wait until bundle finished:
    (node:2188) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
`

- @storybook/react ^3.4.10
- @storybook/html ^4.0.0-alpha.11

**Comment by igor-dv** [comment]
Yeah, every supported framework in storybook is creating the same cli file, so they are overriding each other.

You can try using something like this:

```
"storybook:html": "node node_modules/@storybook/react/bin/index -p 9001 -c .storybook-react",
"storybook:react": "node node_modules/@storybook/html/bin/index -p 8001 -c .storybook-html",
```

Of course, it's a workaround.
